Microsoft.Toolkit.Uwp.UI.Controls.Input/RichSuggestBox/RichSuggestBox.xaml

```diff
@@ -25,7 +25,12 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
-    <Style TargetType="controls:RichSuggestBox">
+    <!--  Default style for RichSuggestBox  -->
+    <Style BasedOn="{StaticResource DefaultRichSuggestBoxStyle}"
+           TargetType="controls:RichSuggestBox" />
+
+    <Style x:Key="DefaultRichSuggestBoxStyle"
+           TargetType="controls:RichSuggestBox">
```

Comment:
@michael-hawker I found the problem. The test will fail if I don't provide a key for the default style. Not exactly sure why that is the case.